server/models/index.ts

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+const config = require( '../config/config');
+const basename = path.basename(__filename);
+const db: any = {};
+
+const sequelize = new Sequelize.Sequelize(config.DB_URL,  {
```

Comment:
Can you update this to include
```
const sequelize = new Sequelize('sqlite::memory:')
```
if we're in test? You should be able to check for `ci` to be true in the environment.

Reply:
Done